src/datasource.ts

```diff
@@ -261,6 +260,14 @@ export default class CogniteDatasource extends DataSourceApi<
     const timeFrame = {
       activeAtTime: { min: start, max: end },
     };
+    const filtered = each(targets, ({ tab }) => tab === Tab.Relationships);
+    if (filtered.length) {
+      return Promise.all(
+        targets.map(async (target) => {
+          return this.createRelationshipsNode(target.relationsShipsQuery, target.refId);
```

Comment:
what is this code doing here? this function is made for events. no relationships are expected here

Reply:
Maybe I didn't understand well the "ask" before to move the code and integrate into somewhere else. I'm looking forward for a better integration.
the 2 "hot" points : `fetchEventTargets && groupTargets` will be reanalyzed and reimplemented in a way to don't conflict with other tabs, and to keep:  the `Tab.Timesseries` as the default, so undefined should appear as timeseries as well.

Reply:
@KissSpinobleZsolt yes, essentially you need your own `fetchRelationshipTargets` that will take `relationshipTargets` result from `groupTargets` call

Reply:
then in the main handler you will only have
```
 const relationshipsResults = await this.fetchRelationshipsTargets(relationshipTargets);
 // ....
 responseData = [...reduceTimeseries(succeded, timeRange), ...eventResults, ...relationshipsResults];
```

Reply:
@polomani yes, I'm hard thinking on it to integrate, it bothers me 2 things. 
1. I have to map array of query's, but I may give back only one array of [nodes, edges] .. here my issue is the reducer function n the end of the calls (I'm hardly trying to fix this issue)
2. at the relationshipslistTab  I'm getting 
      `[ERROR] Parser: Unexpected end of input:
      
      ^` 
    and I don't know from where it is, it happens only if timeseries are also selected.

Reply:
> [ERROR] Parser: Unexpected end of input:

make sure that targets are grouped correctly, maybe `relationshipslist` target somehow appears in assets/events handler

Reply:
Its not about grouping, grouping is good, the problem is the always true relationshipsQuery, if I select a value on a different tab then Relationship the relationship is also called messing the data with unwanted [nodes, edges]. now I fixed it in the call no to be called if the tab is not the right on, in our case the Tab.Relationship